yb-voyager/cmd/analyzeSchema.go

```diff
@@ -774,6 +795,9 @@ func getCreateObjRegex(objType string) (*regexp.Regexp, int) {
 	} else if objType == "TABLE" || objType == "PARTITION" {
 		createObjRegex = re("CREATE", opt("OR REPLACE"), "TABLE", ifNotExists, capture(ident))
 		objNameIndex = 3
+	} else if objType == "FOREIGN TABLE" {
+		createObjRegex = re("CREATE", opt("OR REPLACE"), "FOREIGN", "TABLE", ifNotExists, capture(ident))
```

Comment:
Slightly confused here. We are using both parser as well as regex?

Reply:
No no, this is for a different part of the code where we are using this regex to parse the stmt and get the basic info to prepare the `sqlInfo` object for further processing. I don't want to change this part of the code to parser yet for this case so just added regex.

Reply:
But for reporting we are using Parser.